staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -108,7 +129,11 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr admission.Attr
 		go func(invocation *generic.WebhookInvocation, idx int) {
 			ignoreClientCallFailures := false
 			hookName := "unknown"
-			versionedAttr := versionedAttrs[invocation.Kind]
+			versionedAttr, err := versionedAttrAccessor.VersionedAttribute(invocation.Kind)
+			if err != nil {
+				utilruntime.HandleError(fmt.Errorf("validating webhook dispatch was not able to fetch expected versionedAttributes for %v", invocation.Webhook.GetName()))
```

Comment:
this is not an expected error, but if we error here, it should _not_  just be logged and ignored

rather than logging, send an error with `errCh <- apierrors.NewInternalError(...` (see the example below) before returning

Reply:
putting this return here also means we'll deadlock (because we return before calling wg.Done()) and we're vulnerable to panics crashing the API server (because we're calling this before recover())

maybe just change this to `versionedAttr := versionedAttrAccessor.versionedAttrs[invocation.Kind]` to keep this equivalent to existing code which assumes all needed GVKs have already been allocated

Reply:
got it